src/main/kotlin/interpreter/pyobject/types/PyBytesType.kt

```diff
@@ -23,12 +23,10 @@ import green.sailor.kython.interpreter.iface.ArgType
 import green.sailor.kython.interpreter.iface.PyCallableSignature
 import green.sailor.kython.interpreter.pyobject.*
 import green.sailor.kython.interpreter.throwKy
-import java.lang.IndexOutOfBoundsException
 
 object PyBytesType : PyType("bytes") {
     override fun newInstance(kwargs: Map<String, PyObject>): PyObject {
-        val value = kwargs["value"] ?: error("Built-in signature mismatch")
-        when (value) {
+        when (val value = kwargs["value"] ?: error("Built-in signature mismatch")) {
```

Comment:
Not a fan of this, keep the temporary variable separate - plus having an `?: error(...)` in this block looks really weird.